modules/installation-mirror-repository.adoc

```diff
@@ -55,8 +56,43 @@ the pull secret for your mirror registry that you created.
 `ocp-release`.
 <7> For `server_architecture`, specify the architecture of the server, such as `x86_64`.
 
+. Review the images and configuration manifests to mirror:
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
+----
+
+. Mirror the version images to the internal container registry:
+** If your mirror host does not have internet access, take the following actions:
+... Connect the removable media to a system that is connected to the internet.
+... Mirror the images to a directory on the removable media:
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
+... Take the media to the restricted network environment and upload the images to the local container registry.
++
+----
+$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror 'file://openshift/release:${OCP_RELEASE}*' ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}
+----
++
+[IMPORTANT]
+====
+Because you did not mirror the images directly to your repository, you must manually add the  `imageContentSources` data to the `install-config.yaml` file during installation. You must provide the name of the local registry and repository that you mirrored the images to in the following format:
+
+----
+imageContentSources:
+- mirrors:
+  - <local_registry>/<local_repository_name>/release
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - <local_registry>/<local_repository_name>/release
+  source: registry.svc.ci.openshift.org/ocp/release
+----
+====
 
-. Mirror the repository:
+** If the local container registry and the cluster are connected to the mirror host, take the following actions:
```

Comment:
This is for a fresh install, so there should be no "the cluster are connected to the mirror host" case.